Werewolf for Telegram/Werewolf Control/Handlers/UpdateHandler.cs

```diff
@@ -1253,7 +1253,17 @@ internal static void HandleCallback(CallbackQuery query)
                                     //create a menu out of this
                                     buttons = new List<InlineKeyboardButton>() { new InlineKeyboardCallbackButton(GetLocaleString("All", language), $"groups|{query.From.Id}|{choice}|all") };
                                     buttons.AddRange(variants.OrderBy(x => x).Select(x => new InlineKeyboardCallbackButton(x, $"groups|{query.From.Id}|{choice}|{x}")));
-
+                                    var playersGames = DB.Players.FirstOrDefault(x => x.TelegramId == query.Message.From.Id);
+                                    var gamecount = playersGames?.GamePlayers.Count ?? 0;
+                                    var message = "";
+                                    if (gamecount >= 500 && choice.Equals("English"))
+                                    {
+                                        message = GetLocaleString("WhatVariantVets", language, choice);
+                                        message =
```

Comment:
This is a typo I suppose...? I don't think you really want to overwrite `message`, do you?

Reply:
Oops. My bad. Didn't see carefully its definition